docs/ske/01-kratix/10-configuring-ske/11-via-helm.mdx

```diff
@@ -346,8 +299,15 @@ To upgrade the version of SKE being deployed by the operator, you can edit your
 Kratix instance definition and change the `version` field to the desired
 version.
 
-Upgrades from a version vX.Y.Z to vX.Y+2.Z are not supported. You must upgrade to
-vX.Y+1.Z first.
+The SKE Operator will perform [pre-upgrade checks](../ske-operator#pre-upgrade-checks)
+and [post-upgrade checks](../ske-operator#post-upgrade-checks) as part of the upgrade flow.
```

Comment:
Call out explicitly that [skip checks doesn't work with Helm](https://syntasso.slack.com/archives/C044T9ZFUMN/p1733929239878179)?